kore/app/exec/Main.hs

```diff
@@ -416,6 +425,18 @@ mainWithOptions execOptions = do
     KoreExecOptions { koreSearchOptions } = execOptions
     KoreExecOptions { koreMergeOptions } = execOptions
 
+    handleErrorRewriteLoop
+        :: ErrorRewriteLoop -> Main ExitCode
+    handleErrorRewriteLoop entry = do
+        logEntry entry
+        return $ ExitFailure 1
```

Comment:
We will only need one handler here, if we make sure that our error entries are always wrapped in `SomeEntry`:

```suggestion
    handleSomeEntry
        :: SomeEntry -> Main ExitCode
    handleSomeEntry (SomeEntry entry) = do
        logEntry entry
        return $ ExitFailure 1
```